docs/serving/getting-started-knative-app.md

```diff
@@ -183,3 +183,9 @@ To remove the sample app from your cluster, delete the service record:
 ```shell
 kubectl delete --filename service.yaml
 ```
+
+Alternatively, delete the service record by name:
```

Comment:
I would remove "record"

Reply:
I removed the word "record", is that correct?